src/Native/pkg/dir.props

```diff
@@ -1,51 +1,35 @@
-    <Ubuntu1404NativePath Condition="'$(Ubuntu1404NativePath)' == ''">$(BuildNativePath)</Ubuntu1404NativePath>
-    <Ubuntu1604NativePath Condition="'$(Ubuntu1604NativePath)' == ''">$(BuildNativePath)</Ubuntu1604NativePath>
-    <Ubuntu1610NativePath Condition="'$(Ubuntu1610NativePath)' == ''">$(BuildNativePath)</Ubuntu1610NativePath>
-    <Alpine343NativePath Condition="'$(Alpine343NativePath)' == ''">$(BuildNativePath)</Alpine343NativePath>
+    <LinuxNativePath Condition="'$(LinuxNativePath)' == ''">$(BinDir)Linux.$(PackagePlatform).$(ConfigurationGroup)\Native\</LinuxNativePath>
+    <WinNativeAOTPath Condition="'$(WinNativeAOTPath)' == ''">$(BinDir)Windows_NT.$(PackagePlatform).$(ConfigurationGroup)\Native_aot\</WinNativeAOTPath>
+    <WinNativePath Condition="'$(WinNativePath)' == ''">$(BinDir)Windows_NT.$(PackagePlatform).$(ConfigurationGroup)\Native\</WinNativePath>
+    <RHELNativePath Condition="'$(RHELNativePath)' == ''">$(LinuxNativePath)</RHELNativePath>
+    <DebianNativePath Condition="'$(DebianNativePath)' == ''">$(LinuxNativePath)</DebianNativePath>
+    <Fedora23NativePath Condition="'$(Fedora23NativePath)' == ''">$(LinuxNativePath)</Fedora23NativePath>
+    <Fedora24NativePath Condition="'$(Fedora24NativePath)' == ''">$(LinuxNativePath)</Fedora24NativePath>
+    <OSXNativePath Condition="'$(OSXNativePath)' == ''">$(BinDir)OSX.$(PackagePlatform).$(ConfigurationGroup)\Native\</OSXNativePath>
+    <OpenSuse132NativePath Condition="'$(OpenSuse132NativePath)' == ''">$(LinuxNativePath)</OpenSuse132NativePath>
+    <OpenSuse421NativePath Condition="'$(OpenSuse421NativePath)' == ''">$(LinuxNativePath)</OpenSuse421NativePath>
+    <Ubuntu1404NativePath Condition="'$(Ubuntu1404NativePath)' == ''">$(LinuxNativePath)</Ubuntu1404NativePath>
+    <Ubuntu1604NativePath Condition="'$(Ubuntu1604NativePath)' == ''">$(LinuxNativePath)</Ubuntu1604NativePath>
+    <Ubuntu1610NativePath Condition="'$(Ubuntu1610NativePath)' == ''">$(LinuxNativePath)</Ubuntu1610NativePath>
+    <Alpine343NativePath Condition="'$(Alpine343NativePath)' == ''">$(LinuxNativePath)</Alpine343NativePath>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
   </PropertyGroup>
 
   <!-- Add path globs specific to native binaries to exclude unnecessary files from packages. -->
-  <ItemGroup Condition="'$(OSGroupPath)'=='Linux' OR '$(OSGroupPath)'=='OSX'">
-    <AdditionalLibPackageExcludes Include="%2A%2A\%2A$(SymbolFileExtension)" />
-    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.a;%2A%2A\%2A$(LibraryFileExtension)" />
+  <ItemGroup>
+    <SymbolFileExtensions Include=".dbg" />
+    <SymbolFileExtensions Include=".dwarf" />
+    <AdditionalLibPackageExcludes Include="@(SymbolFileExtensions->'%2A%2A\%2A%(Identity)')" />
+
+    <LibFileExtensions Include=".a" />
+    <LibFileExtensions Include=".so" />
+    <LibFileExtensions Include=".dylib" />
+    <AdditionalSymbolPackageExcludes Include="@(LibFileExtensions->'%2A%2A\%2A%(Identity)')" />
```

Comment:
Have you confirmed if this still creates native symbol packages? I would think that https://github.com/dotnet/buildtools/blob/dd54797fc686939b7cd8794ff8cdf7e258a7de0d/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs#L185-L189 would skip it, because it expects >= 1 file to exist for every exclude glob.